### PR TITLE
Rename boolean selects

### PR DESCRIPTION
**Pull request by alinauroz**
for https://github.com/cobudget/cobudget/issues/913

### CODE REVIEW

File: ui/components/RoundSettings/Granting/index.tsx [hunk 1]
```diff
@@ -204,7 +204,7 @@ const RoundSettingsModalGranting = ({ currentGroup }) => {
                 bucketName: process.env.BUCKET_NAME_PLURAL,
               }
             )}
-            secondary={round.requireBucketApproval?.toString() ?? "false"}
+            secondary={round.requireBucketApproval ? "Yes" : "No"}
```
Comment (gustavlrsn) [inline ui/components/RoundSettings/Granting/index.tsx:207]: Same here

File: ui/components/About/index.tsx [hunk 2]
```diff
@@ -87,7 +87,7 @@ export default function AboutPage({ router, round }) {
           <ListItem>
             <ListItemText
               primary="Allow stretch goals"
-              secondary={round.allowStretchGoals?.toString() ?? "false"}
+              secondary={round.allowStretchGoals ? "Yes" : "No"}
```
Comment (gustavlrsn) [inline ui/components/About/index.tsx:90]: Make these internationalized

**Review by gustavlrsn — CHANGES_REQUESTED** [review]
Looks good, just some strings that need to be internationalized.

File: ui/components/RoundSettings/Granting/index.tsx [hunk 3]
```diff
@@ -185,7 +185,7 @@ const RoundSettingsModalGranting = ({ currentGroup }) => {
             primary={intl.formatMessage({
               defaultMessage: "Allow stretch goals",
             })}
-            secondary={round.allowStretchGoals?.toString() ?? "false"}
+            secondary={round.allowStretchGoals ? "Yes" : "No"}
```
[the remark this answers is not in the document]
Comment (alinauroz) [reply ui/components/RoundSettings/Granting/index.tsx]: done

File: ui/components/RoundSettings/Granting/index.tsx (hunk 1 above)
Comment (alinauroz) [reply ui/components/RoundSettings/Granting/index.tsx]: done

File: ui/components/About/index.tsx (hunk 2 above)
Comment (alinauroz) [reply ui/components/About/index.tsx]: done

**Pull request merged**